### PR TITLE
Feature/planet cash dashboard section

### PR DESCRIPTION
**Pull request by mohitb35**
1. Adds PlanetCash section to dashboard and moves Bulk Codes into the PlanetCash menu as a submenu item.
2. Logged in user can see a list of PlanetCash accounts (if created), or create a new PlanetCash account.
3. Logged in user can see a list of PlanetCash transactions.
4. Logged in user can deactivate an active PlanetCash account.
5. Logged in user can activate a deactivated PlanetCash account, if no other active account is present.
6. A single account can be created for each of the following countries - US, DE, ES. Once all 3 are created, further accounts cannot be created. 
7. The created account will be automatically activated if no other account is currently active. Otherwise, the user needs to activate the account after account creation.

### CODE REVIEW

**Comment by sagararyal** [comment]
#### 📝 Changed routes:


| [**/profile/planetcash**](https://planet-webapp-m2istcwg0-planetapp.vercel.app/profile/planetcash) |
|:-:|
|<a href="https://planet-webapp-m2istcwg0-planetapp.vercel.app/profile/planetcash">Screenshot of /profile/planetcash</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/NfBlqOmNlmhXK-AGOcVK8oWDZ00j_r9Pd-RjVysOAb_Nx97jpJpWGRFCZaiNgnvVKOF5XeuiElH6fQImX9-pKA.png">(view full size)</a> |







Commit 3c556210c8b670e0b4182850d1ce0021af0e190e (https://planet-webapp-m2istcwg0-planetapp.vercel.app).

File: src/features/common/InputTypes/AutoCompleteCountryNew.tsx [hunk 1]
```diff
@@ -0,0 +1,379 @@
+/* eslint-disable no-use-before-define */
```
Comment (norbertschuler) [inline src/features/common/InputTypes/AutoCompleteCountryNew.tsx:1]: Does the name `AutoCompleteCountryNew.tsx` should imply the old component `AutoCompleteCountry.tsx` should be replaced or should both be used next to each other? In the later case I would wonder if`AutoCompleteCountryNew.tsx` maybe could be renamed to better reflect the purpose of its usage?

File: src/features/user/PlanetCash/screens/CreateAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { usePlanetCash } from '../../../common/Layout/PlanetCashContext';
+import { CountryType } from '../../../common/types/country';
+import i18next from '../../../../../i18n';
+
+const { useTranslation } = i18next;
+
+const initialAllowedCountries: CountryType[] = [
+  { code: 'DE', label: 'Germany', phone: '49' },
+  { code: 'ES', label: 'Spain', phone: '34' },
+  { code: 'US', label: 'United States', phone: '1' },
```
Comment (norbertschuler) [inline src/features/user/PlanetCash/screens/CreateAccount.tsx:13]: Would be nice to have the labels translated with i18n. We have localized country names in `country.json` - all other places where country names are placed in the code only in English should be deprecated and hopefully not used in the UI, e.g.

```javascript
const { t } = useTranslation(['country']);
{ code: 'US', label: t('country:us'), phone: '1' },
```
Code not tested, maybe the const must be initialized more dynamically.

**Review by norbertschuler — CHANGES_REQUESTED** [review]

File: src/features/common/InputTypes/AutoCompleteCountryNew.tsx (hunk 1 above)
Comment (mohitb35) [reply src/features/common/InputTypes/AutoCompleteCountryNew.tsx:1]: We're updating the dashboard UI page by page. `AutoCompleteCountryNew.tsx` is the component with the new UI and `AutoCompleteCountry.tsx` is the old one - `AutoCompleteCountryNew` will eventually replace `AutoCompleteCountry` in all places, and then will be renamed to `AutoCompleteCountry` (once the old component is no longer useful).

File: src/features/user/PlanetCash/screens/CreateAccount.tsx (hunk 2 above)
Comment (mohitb35) [reply src/features/user/PlanetCash/screens/CreateAccount.tsx]: We're not using label anywhere that is visible to the user. I've updated this to remove label from the initialCountries array.

**Review by norbertschuler — APPROVED** [review]
LGFM

**Pull request merged**